src/components/PressableWithSecondaryInteraction/pressableWithSecondaryInteractionPropTypes.js

```diff
@@ -43,6 +43,7 @@ const defaultProps = {
     preventDefaultContentMenu: true,
     inline: false,
     withoutFocusOnSecondaryInteraction: false,
+    isLongPressEnabledWithHover: false,
```

Comment:
NAB: Is `...withHover` necessary here?

Reply:
More specifically, I meant that this prop name is quite long and didn't make sense to me immediately.